### PR TITLE
Updates for jikanpy_v4 release

### PR DESCRIPTION
**Pull request by Chris-Peterson444**
Addressing #103 I've gone ahead and updated a few things in getting ready to push the new package: 

### New Package Name: `jikanpy_v4`
As discussed in #103, there are potentially people who still rely on jikanpy for functionality on self-hosted versions of Jikan v3. So the plan is to publish the updated version as `jikanpy_v4` on pypi. The old version will be untouched and the source code has been moved to the branch `jikanpy_v3`. 

### New Changelog
Since `jikanpy_v4` will be its own package I decided we should start over with the changelog. The history could be nice to have, but considering the version numbers won't match I thought it would be confusing. That being said, I figure version 1.0.0 is a good place to start.

### Misc README Updates
All of the links should be updated correctly and new install install instructions have been included.

### Authorship and License
@abhinavk99 what do you think of putting myself as the author for this when pushing to pypi?  I've also included myself on the license file, if this is alright with you.

### Docs
The last thing we probably want to consider is the readthedocs page. Currently, I believe it reads from the master branch although I'm not entirely sure how it works. Perhaps we want to make separate pages for the new and old versions? 

If you approve of these changes @abhinavk99 I'll go ahead and merge to master and push to pypi.

### CODE REVIEW

**Review by abhinavk99 — APPROVED** [review]
LGTM! About the docs, it’s been a while so I don’t remember exactly how it works and I can’t confirm currently because I’m on my phone on vacation, but it should just read from master as the CI pipeline looks like it’s still running as normal.

**Comment by BisericaOrtodoxa** [comment]
When will jikanpy_v4 be on pypi? Can't currently find it

File: README.md [hunk 1]
```diff
@@ -25,19 +25,36 @@ are below.
...
 
 ## Installation
+You can install the package from PyPI using pip:
 
 ```shell
-$ pip install jikanpy
+$ pip install jikanpy-v4
 ```
 
+If you have previously installed the old version of jikanpy, then you make sure to uninstall the old version first:
```
[the remark this answers is not in the document]
Comment (Chris-Peterson444) [reply README.md]: Ah whoops. Good catch, thanks!

**Comment by Chris-Peterson444** [comment]
@BisericaOrtodoxa I should be able to have it up today. Workin on it now.

**Pull request merged**